docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
         NODE_ENV: development
     command: yarn run start-watch
     environment:
-      API_BASE_URL: 'http://nearby-services-api:3001'
+      API_BASE_URL: '${API_BASE_URL-http://nearby-services-api:3001}'
```

Comment:
is the env var `API_BASE_URL` used in local development?

Reply:
I have been using it so I can point at the latest version of the API.